website/content/maintainers/templates/_index.md

```diff
@@ -4,50 +4,62 @@ linkTitle: "Templates"
 layout: single
 date: 2020-11-07
 description: >
-  Jump start your CNCF project by using our recommended templates
+  Jump start your CNCF project by using our templates
+aliases:
+- /maintainers/github/templates/
 ---
 
 We have a [project template] repository that has templates for everything your project needs
-to get started and eventually join the CNCF. You can either create a new repository using the repository template or copy just the templates that you require.
+to get started and eventually join the CNCF. 
 
 ## Getting Started
 
-1. Click **Use this template** and create a copy of this repository.
+The templates are stored in the [project-template] repo,
+but we also have how-to guides here on the website that are a better place to
+start if you haven't used a particular template before.
 
-    ![Green button that says "Use this template"](https://user-images.githubusercontent.com/1368985/95903529-e9c32f00-0d5b-11eb-8723-4369f7c9e044.png)
-1. Remove **.github/settings.yml**. This is not a template and contains
-   configuration specific our repository. You should not keep this file.
-1. Customize every [required file](#required-files) and address each TODO item.
-1. Security templates may be found in the [Security TAG's project resources
-   folder](https://github.com/cncf/tag-security/tree/main/project-resources).
+Please note that the security templates may be found in the
```

Comment:
This sentence used to make more sense because it was in the context of cloning the template repository (and the security templates are in a different repo).

Now that we aren't telling people clone the repo and instead they are finding templates from the list further below on the page (which includes the security templates), what do you think about removing this sentence?